.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,26 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
```

Comment:
This is a bit new to me, does this mean it will run once for each version of node? Are we using 8.x or 10.x in places?

Reply:
we are definitely using 10.x for some lambdas. And I think some older lambdas might still be using 8.x, but we could maybe drop that one.

Reply:
It's probably fine to leave it in for now - it won't be running that often anyway. I was more just curious about what we were running

Reply:
As this is an open source project it is free anyway :D

Reply:
Actually, the lambdas only use eslint-config-fishbrain-base btw, so I guess that doesn't really apply to this package.